chapters/10-community-involvement.Rmd

```diff
@@ -129,6 +156,11 @@ step, refer to the
     exercise, then add something to CONTRIBUTING.md about what the leadership
     structure is, if any, and how to get there.
 
+5. Project management
```

Comment:
Sorry, forgot to remove the Project Governance item above during merge. You can remove it again.